docs/DeveloperGuide.md

```diff
@@ -283,32 +283,65 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `Staff'd` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 - Edit staff details**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User chooses to edit the staff details.
```

Comment:
Can change all the numbers to 1 following GFMD standard.

Reply:
Change all the numbers in the steps to 1?

Reply:
I was thinking of something like this.

![image](https://user-images.githubusercontent.com/70463178/134852036-324ac162-47b4-42cf-8009-cf9a3b7bc879.png)

Reply:
Hmm this part is the steps in the use case tho, I don't think it makes sense to have them all be 1 haha, anyway I went back to see the use case in ab3 and it was numbered similarly too. Unless I'm interpreting this wrongly and you mean something else?

Reply:
Should be fine. Just thought it might make it easier to edit the MSS later on.

Reply:
I also believe it should be listed normally. The use case example given by AB3 isn't listed with all 1s. 
![image](https://user-images.githubusercontent.com/69857486/134868537-28f660f4-d00d-4cd1-b00c-39fddfc20d12.png)

Reply:
GFMD formats all 1s in a list to ordered.

Reply:
Hmm ya but for the extension part like 3a1 it doesn't tho so I thought something more consistent would be better, we can do both tho I guess

Reply:
Okie nvm I have no strong feelings about this I guess I've updated it